### PR TITLE
Move o.e.tips.examples to Java 17

### PR DESCRIPTION
**Pull request by vrubezhny**

### CODE REVIEW

**Review by akurtakov — CHANGES_REQUESTED** [review]

File: org.eclipse.tips.examples/src/org/eclipse/tips/examples/browserfunction/BrowserFunctionTip.java [hunk 1]
```diff
@@ -40,10 +40,11 @@ public Map<String, Function<Object[], Object>> getBrowserFunctions() {
...
 	@Override
 	public String getHTML() {
-		return "<html><head><title>IHtmlTip with IBrowserFunctionProvider</title></head>"
-				+ "<body><p>This tip shows HTML and provides a BrowserFunction, which can be invoked by using JavaScript.</p>"
-				+ "<p><button onclick=\"openPreferencesBrowserFunction()\">I gonna open the preferences from the Browser</button></p></body></html>"
-				+ "<p><a href=\"#\" onclick=\"openPreferencesBrowserFunction()\">I do the same but with a link</a></p></body></html>";
+		return """
+				<html><head><title>IHtmlTip with IBrowserFunctionProvider</title></head>
+				<body><p>This tip shows HTML and provides a BrowserFunction, which can be invoked by using JavaScript.</p>
+				<p><button onclick=\"openPreferencesBrowserFunction()\">I gonna open the preferences from the Browser</button></p></body></html>
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply org.eclipse.tips.examples/src/org/eclipse/tips/examples/browserfunction/BrowserFunctionTip.java:46]: ACK

File: org.eclipse.tips.examples/src/org/eclipse/tips/examples/eclipsetips/TwitterTip.java [hunk 2]
```diff
@@ -38,9 +38,10 @@ public String getSubject() {
...
-				+ "<script src=\"https://platform.twitter.com/widgets.js\" charset=\"utf-8\"></script></div></body></html>";
+		return """
+				<html><head><style>
+				</style></head><body><div><blockquote class=\"twitter-tweet\" data-lang=\"en\"><p lang=\"en\" dir=\"ltr\">The &#39;Extract class...&#39; refactoring (from Alt+Shift+T) extracts a group of fields into a separate class and replaces all occurrences to fit the new structure. See example. <a href=\"https://twitter.com/hashtag/EclipseTips?src=hash&amp;ref_src=twsrc%5Etfw\">#EclipseTips</a> <a href=\"https://t.co/tEI7ic7C1g\">pic.twitter.com/tEI7ic7C1g</a></p>&mdash; Eclipse Java IDE (@EclipseJavaIDE) <a href=\"https://twitter.com/EclipseJavaIDE/status/949238007051235328?ref_src=twsrc%5Etfw\">January 5, 2018</a></blockquote>
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply org.eclipse.tips.examples/src/org/eclipse/tips/examples/eclipsetips/TwitterTip.java:43]: ACK

File: org.eclipse.tips.examples/src/org/eclipse/tips/examples/java/java9/Tip1.java [hunk 3]
```diff
@@ -85,10 +85,11 @@ public String getSubject() {
 	 */
 	@Override
 	public String getHTML() {
-		return "<h2>Javatip 1</h2>You see this tip because the Tip UI was opened when the java perspective "
-				+ "was active or because you selected the Java icon below." + "<br><br>"
-				+ "More java tips will be displayed here in the near future. For now, select one of the other providers"
-				+ " by clicking on the icons below.";
+		return """
+				<h2>Javatip 1</h2>You see this tip because the Tip UI was opened when the java perspective
+				was active or because you selected the Java icon below." + "<br><br>
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply org.eclipse.tips.examples/src/org/eclipse/tips/examples/java/java9/Tip1.java:90]: ACK

File: org.eclipse.tips.examples/src/org/eclipse/tips/examples/tipsframework/WelcomeTip.java [hunk 4]
```diff
@@ -38,9 +38,10 @@ public String getSubject() {
 
 	@Override
 	public String getHTML() {
-		return "<h2>Welcome to the Tips Framework</h2>It can show tips from various tip providers. This provider has tips about tips which will show you how to navigate this UI."
-				+ " The dialog is this Tip UI. Tips appear here in various forms. They can come from Twitter, a Wiki, a Website, a file or even from Java, like this one."
-				+ "<br><br>" + "Press <b><i>Next Tip</i></b> to see how to start tips manually.";
+		return """
+				<h2>Welcome to the Tips Framework</h2>It can show tips from various tip providers. This provider has tips about tips which will show you how to navigate this UI.
+				 The dialog is this Tip UI. Tips appear here in various forms. They can come from Twitter, a Wiki, a Website, a file or even from Java, like this one.
+				<br><br>" + "Press <b><i>Next Tip</i></b> to see how to start tips manually.""";
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply org.eclipse.tips.examples/src/org/eclipse/tips/examples/tipsframework/WelcomeTip.java:44]: ACK

File: org.eclipse.tips.examples/src/org/eclipse/tips/examples/tipsframework/StartingTip.java [hunk 5]
```diff
@@ -40,9 +40,10 @@ public String getSubject() {
 
 	@Override
 	public String getHTML() {
-		return "<h2>Opening the Tips Dialog</h2>The tips are started automatically at startup but you can switch this off."
-				+ " In case the tips are not loaded at startup you can active the tips manually from the Help menu."
-				+ "<br><br>" + "Press <b><i>Next Tip</i></b> to see how to navigate Tips.<br><br>";
+		return """
+				<h2>Opening the Tips Dialog</h2>The tips are started automatically at startup but you can switch this off.
+				 In case the tips are not loaded at startup you can active the tips manually from the Help menu.
+				<br><br>" + "Press <b><i>Next Tip</i></b> to see how to navigate Tips.<br><br>""";
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply org.eclipse.tips.examples/src/org/eclipse/tips/examples/tipsframework/StartingTip.java:46]: ACK

**Review by akurtakov — APPROVED** [review]

**Pull request merged**